examples/Tutorial.ipynb

```diff
@@ -0,0 +1,925 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7d1dcb07",
+   "metadata": {},
+   "source": [
+    "# Piff Tutorial\n",
+    "\n",
+    "### Links\n",
```

Comment:
Maybe at the top, you could add a section/header like the ones in the DP0.2 tutorials, which include information such as the version of the code used to run the notebook ad the last time it was verified to run: https://github.com/rubin-dp0/tutorial-notebooks/blob/main/01_Introduction_to_DP02.ipynb

Reply:
It's being run as part of the CI tests, and it's hosted in the PIff repo itself, so it will be kept current with the current version of Piff, and there will be a version connected to each minor release series, which would be known to work with that version.